### DISCUSSION TITLE
How to clean historic metadata ?

### DISCUSSION THREAD

**Discussion by nitra-1** [apache/airflow#25116] [General]
I am using PG as Airflow metadata store, it is eating too much of disk space.
How to clean historic metadata, like i want to keep data for two months only.

**Comment by alexkruc** [comment]
I can suggest 2 ways- 
1. Use the `airflow db cleanup` command, here is the documentation:
https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#clean

2. Use the maintenance dags from Team Clairvoyant, they have a `db cleanup dag`  that you can schedule from your own Airflow
https://github.com/teamclairvoyant/airflow-maintenance-dags
